docs/components/tfidf_ranking.rst

```diff
@@ -17,42 +17,52 @@ Config
 ======
 
 Default ranker config for **English** language is
-:config:`doc_retrieval/en_ranker_tfidf_wiki.json <ranking/en_ranker_tfidf_wiki.json>`
+:config:`doc_retrieval/en_ranker_tfidf_wiki.json <doc_retrieval/en_ranker_tfidf_wiki.json>`
 
 Default ranker config for **Russian** language is
-:config:`doc_retrieval/ru_ranker_tfidf_wiki.json <ranking/ru_ranker_tfidf_wiki.json>`
+:config:`doc_retrieval/ru_ranker_tfidf_wiki.json <doc_retrieval/ru_ranker_tfidf_wiki.json>`
 
 Config Structure
 ----------------
```

Comment:
We already have "Configuration files" doc page describing the general config structure. Consider writing your doc page assuming the user knows that; just add some reference to the "Configuration files" and corresponding API doc.

Reply:
ok